A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -316,6 +316,51 @@ class FMCMainDisplay extends BaseAirliners {
         this._inOutElement.style.color = color;
     }
 
+    /**
+     * Returns true if an engine is not running (FF === 0)
+     * @returns {number}
+     */
+    isAnEngineOff() {
+        return Simplane.getEngineActive(0) && Simplane.getEngineActive(1);
```

Comment:
shouldn't this be an or? looks like it checks if both engines are active

Reply:
Yes to match the name it should be, the method should be called `areBothEnginesOn`

Reply:
Fixed

Reply:
doesn't the FCOM state that an engine needs to be one, not both?

Reply:
this is used to check whether an engine fail or happened, this is a condition whether to shop OPT FL or not.